src/test/java/org/selenium/pages/tests/RegistrationPageTest.java

```diff
@@ -0,0 +1,37 @@
+package org.selenium.pages.tests;
+
+import io.qameta.allure.Epic;
+import io.qameta.allure.Step;
+import jdk.jfr.Description;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.selenium.utils.ListenerEvent;
+
+import static org.junit.Assert.assertEquals;
+
+@ExtendWith(ListenerEvent.class)
+public class RegistrationPageTest extends BaseTest {
+    @BeforeEach
+    @Step("Beginning the action")
+    public void getStart() {
+        mainPage.logIn();
+        signIn.registration();
+    }
```

Comment:
If you have only one test, do you need before ?

Reply:
Fixed it